src/com/algosolution/AlgoSolution.java

```diff
@@ -0,0 +1,23 @@
+package com.algosolution;
+
+
+
+public class AlgoSolution {
+    public static void FIND(int Start, int End, String Text) {
+        String[] MonTableau = Text.split("");
+
+        for(int i = 0; i < MonTableau.length; ++i) {
+            if (i >= Start && i <= End) {
+                MonTableau[i] = "";
+            }
+
+            System.out.print(MonTableau[i]);
```

Comment:
I said your function should return i didn't you should print. So return your result here please not print
```public static String FIND(int start, int end, String text)```